### PR TITLE
Update Incoming Funds section

### LINKED ISSUE #3312: Moving Claiming Incoming Funds and new Incoming Funds aside
### Story

If funds are sent to a colony or tokens are minted with a motion, you need to manually claim those funds within the Events page of a colony. Without knowing this and without navigating to the events page to claim funds, it would seem as though those funds had not arrived in the colony. This can be quite confusing and frustrating.

### Description

Currently, the claiming of incoming funds is accessible only from ‘Events’ page. However, with the Events page being moved and considering the story above, claiming funds will be moved to a more prominent location, this being the right hand aside, as well as on the ‘Funds’ page.

In both locations it will work similar to how it currently does on the ‘Events’ page, where it is only visible when there are incoming funds to claim.

- The title of the section in the aside will link to the funds page.
- The claim button in the aside will trigger the claim for all tokens, this will likely require updates to the claim saga.
- The aside will only show one of the incoming fund tokens, the the tally underneath as of the design.
- The incoming funds on the funds page, can show it as it is currently is on the events page.

### Design

[Figma link](https://www.figma.com/file/qQoBbyyDXkuHwoqHIOGjEu/Colony-Simplified?node-id=6452%3A82853)

new-claiming-funds-3
new-claiming-funds-1
new-claiming-funds-2

### This issue is complete when...

- [x] The claiming incoming funds component has been moved to the funds page.
- [x] The new claiming funds component has been added to the aside.
- [x] The ‘Claim’ button in the aside is working for multiple token claims.
- [x] The tooltip has been added to the 'Claim' button.

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
This is the feature branch for the new "Incoming funds" sections in the Funds and the aside on the Home page.

163577647-b1407a10-dfb1-43ab-9949-8e75ae038b93 (1)
163577652-ab62f572-d410-4ec3-8a66-d7124ddb0d9f (1)

Resolves #3312

### CODE REVIEW

**Comment by rdig** [comment]
What's the state of this? Are there any more PRs or Issues left in this feature ?

**Comment by chinins** [comment]
> What's the state of this? Are there any more PRs or Issues left in this feature ?

There is my PR that's left and I will submit it for review today. Hopefully, can merge by Monday.

File: src/modules/dashboard/components/UnclaimedTransfers/UnclaimedTransfersItem.tsx [hunk 1]
```diff
@@ -1,30 +1,18 @@
...
+import { useTokenQuery, ColonyTransaction, useLoggedInUser } from '~data/index';
 import { ActionTypes } from '~redux/index';
 import { mergePayload } from '~utils/actions';
 import { getTokenDecimalsWithFallback } from '~utils/tokens';
 import { checkIfNetworkIsAllowed } from '~utils/networks';
 
-import { tokenIsETH } from '../../../core/checks';
-
 import styles from './UnclaimedTransfersItem.css';
+import CopyableAddress from '~core/CopyableAddress';
```
Comment (danbr) [inline src/modules/dashboard/components/UnclaimedTransfers/UnclaimedTransfersItem.tsx:15]: shouldnt this import be located in the block above?

**Review by danbr — APPROVED** [review]
Looks good - well done.

**Review by danbr — CHANGES_REQUESTED** [review]
I was testing by minting tokens using a motion. I created several motions.
I did a lot of playing...

Generally, i need to refresh the page manually to make Incoming funds text appear or disappear.

Otherwise i can claim a finalised motion (on motion page), and then go to action page and the funds are still incoming, also the same when i go to manage funds.
I can (again) claim the funds, navigate back and forth and the display is not updated.
If i manually refresh the browser the incoming fund message is removed.

On the manage funds page, If I click the Claim for colony, the transaction is processed but the page/list is not updated. If i refresh, then all incoming funds disappear, i assume they have all been claimed.

Screenshot 2022-06-07 at 12 54 47


If this sounds confusing, i'm happy to have a call and go through it......


Looks like the amount & token symbol are not aligned.
Screenshot 2022-06-07 at 12 29 13

Screenshot 2022-06-07 at 12 14 24


More margin than figma
Screenshot 2022-06-07 at 12 16 44

File: src/modules/dashboard/components/UnclaimedTransfers/UnclaimedTransfersItem.tsx (hunk 1 above)
Comment (ArmandoGraterol) [reply src/modules/dashboard/components/UnclaimedTransfers/UnclaimedTransfersItem.tsx:15]: You are correct as always Mr. Dan

**Comment by ArmandoGraterol** [comment]
@danbr Thank you so much for testing! I appreciate the feedback.

> Generally, i need to refresh the page manually to make Incoming funds text appear or disappear.
> 
> Otherwise i can claim a finalised motion (on motion page), and then go to action page and the funds are still incoming, also the same when i go to manage funds. I can (again) claim the funds, navigate back and forth and the display is not updated. If i manually refresh the browser the incoming fund message is removed.
> 
> On the manage funds page, If I click the Claim for colony, the transaction is processed but the page/list is not updated. If i refresh, then all incoming funds disappear, i assume they have all been claimed.

Yeah, we encountered this same problem during the live QA session. Unfortunately, this issue is something that was carried from the previous version of the incoming funds section, and the solutions, as of now,  are not at all straightforward or simple (Raul for example mentioned that one solution could be implementing subscriptions to retrieve the data) and may be too much for this project in particular and the time we have available to work on it.

So, this particular issue will need to be tackled later, on a separate branch from this one.

**Review by arrenv — APPROVED** [review]
Great work!

**Review by danbr — APPROVED** [review]
LGTM - well done..... tip-top!!

**Review by chinins — APPROVED** [review]
Very nice! I think this is much more user friendly :)

**Pull request merged**